### PR TITLE
Bat-native-ledger linter fix (part 3)

### PR DESCRIPTION
**Pull request by NejcZdovc**
Resolves https://github.com/brave/brave-browser/issues/3413

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Windows
  - [ ] macOS
  - [ ] Linux
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on
  - [ ] Windows
  - [x] macOS
  - [ ] Linux
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

### CODE REVIEW

File: utility/importer/brave_importer.cc [hunk 1]
```diff
@@ -39,7 +39,8 @@
 #include "sql/statement.h"
 #include "url/gurl.h"
 
-#include "brave/vendor/bat-native-ledger/src/static_values.h"
+#include \
```
Comment (bridiver) [inline utility/importer/brave_importer.cc]: this is incorrect, it should just be "bat/ledger/internal/static_values.h"

File: components/brave_rewards/browser/rewards_service_browsertest.cc [hunk 2]
```diff
@@ -8,8 +8,8 @@
 #include "brave/common/brave_paths.h"
 #include "brave/components/brave_rewards/browser/rewards_service_impl.h"
 #include "brave/components/brave_rewards/browser/rewards_service_factory.h"
-#include "brave/vendor/bat-native-ledger/src/bat_helper.h"
-#include "brave/vendor/bat-native-ledger/src/static_values.h"
+#include "brave/vendor/bat-native-ledger/src/bat/ledger/internal/bat_helper.h"
```
Comment (bridiver) [inline components/brave_rewards/browser/rewards_service_browsertest.cc]: we shouldn't ever be using `brave/vendor/bat-native-ledger/src/` in the header path. The test target should have `bat-native-ledger:internal_config` so it has the correct include paths. Using the full path like this always indicates that you have a gn issue

File: vendor/bat-native-ledger/src/bat/ledger/internal/test/niceware_partial_unittest.cc [hunk 3]
```diff
@@ -4,7 +4,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "brave/components/brave_rewards/resources/grit/brave_rewards_resources.h"
-#include "brave/vendor/bat-native-ledger/src/bat_helper.h"
+#include "brave/vendor/bat-native-ledger/src/bat/ledger/internal/bat_helper.h"
```
Comment (bridiver) [inline vendor/bat-native-ledger/src/bat/ledger/internal/test/niceware_partial_unittest.cc]: see other comments about `brave/vendor`

File: components/brave_rewards/browser/rewards_service_browsertest.cc (hunk 2 above)
Comment (NejcZdovc) [reply components/brave_rewards/browser/rewards_service_browsertest.cc]: fixed

File: vendor/bat-native-ledger/src/bat/ledger/internal/test/niceware_partial_unittest.cc (hunk 3 above)
Comment (NejcZdovc) [reply vendor/bat-native-ledger/src/bat/ledger/internal/test/niceware_partial_unittest.cc]: fixed

File: utility/importer/brave_importer.cc (hunk 1 above)
Comment (NejcZdovc) [reply utility/importer/brave_importer.cc]: done

File: components/brave_rewards/resources/ui/internals/brave_rewards_internals.tsx [hunk 4]
```diff
@@ -17,12 +17,12 @@ import * as rewardsInternalsActions from './actions/rewards_internals_actions'
 window.cr.define('brave_rewards_internals', function () {
   'use strict'
 
-  function getRewardsEnabled() {
+  function getRewardsEnabled () {
```
Comment (NejcZdovc) [inline components/brave_rewards/resources/ui/internals/brave_rewards_internals.tsx]: fixed current travis lint errors

File: components/brave_rewards/resources/ui/internals/brave_rewards_internals.tsx [hunk 5]
```diff
@@ -17,12 +17,12 @@ import * as rewardsInternalsActions from './actions/rewards_internals_actions'
...
     const actions = bindActionCreators(rewardsInternalsActions, store.dispatch.bind(store))
     actions.getRewardsEnabled()
   }
 
-  function getRewardsInternalsInfo() {
+  function getRewardsInternalsInfo () {
```
Comment (NejcZdovc) [inline components/brave_rewards/resources/ui/internals/brave_rewards_internals.tsx]: fixed current travis lint errors

**Pull request merged**